HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json

```diff
@@ -30,34 +66,660 @@
           "valueCanonical": "questionnaire/patient-info"
         }
       ],
-      "type": "display",
-      "text": "This is a placeholder for Patient/Beneficiary Demographics"
+      "type": "display"
     },
     {
       "linkId": "2",
+      "text": "Additional Patient Information",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "2.1",
+          "text": "Address",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".Line"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "2.2",
+          "text": "City",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".City"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "2.3",
+          "text": "State",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".State"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "2.4",
+          "text": "Zip",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".Zip"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "2.5",
+          "text": "Telephone",
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPatientInfoPrepopulation\".Phone"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "text": "Provider (physician/allowed NPP) who performed the face-to-face examination (see guidance)",
+      "type": "group",
+      "item": [
```

Comment:
Use encounter-provider subquestionnaire instead, then use expand to add additional items.